client/go/cmd/test.go

```diff
@@ -0,0 +1,357 @@
+// Copyright Yahoo. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+// vespa test command
+// Author: jonmv
+
+package cmd
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"github.com/spf13/cobra"
+	"github.com/vespa-engine/vespa/client/go/util"
+	"github.com/vespa-engine/vespa/client/go/vespa"
+	"io"
+	"io/ioutil"
+	"math"
+	"net/http"
+	"net/url"
+	"os"
+	"path"
+	"strings"
+	"time"
+)
+
+func init() {
+	rootCmd.AddCommand(testCmd)
+}
+
+var testCmd = &cobra.Command{
+	Use:   "test [tests directory or test file]",
+	Short: "Run a test suite, or a single test",
+	Long: `Run a test suite, or a single test
+
+Runs all JSON test files in the specified directory, or the single JSON
```

Comment:
I assume we'll eventually have a documentation page for this feature. We should link to that here when it's added.

Reply:
Good idea, will add a TODO for this.